userguide/aviate/how-to-install-the-aviate-plugin.adoc

```diff
@@ -0,0 +1,136 @@
+= How to Install the Aviate Plugin
+
+include::{sourcedir}/aviate/includes/aviate-card.adoc[]
+
+//TODO - Add some introduction about Aviate plugin
+
+== Prerequisites
+
+* Ensure that you have Kill Bill, Kaui, and the database set up as explained in the https://docs.killbill.io/latest/getting_started.html[__Getting Started Guide__].
+
+* Ensure that you have https://curl.haxx.se/[_cURL_] installed. If you are on Windows, we recommend that you use https://git-scm.com/download/win[_Git Bash_] to run the `cURL` commands.
+
+== Plugin Installation
+
+The Aviate plugin is not an open-sourced plugin and hence cannot be installed using the standard https://docs.killbill.io/latest/plugin_installation#_plugin_installation[__plugin installation steps__].
+
+This section lists the steps to be followed for the Aviate plugin installation.
+
+//TODO - Steps to create the DB tables are not mentioned here, I'm assuming that com.killbill.billing.plugin.aviate.enableMigrations=true will automatically create the tables?
```

Comment:
Yes, they should. We should explain how the flyway migration process works and also share the DDL somewhere (https://docs.killbill.io/latest/aviate-mysql.ddl ?)

Reply:
I'll gather more information about this and address this in a subsequent PR.